dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/dataaccess/FileAccessIO.java

```diff
@@ -118,10 +118,10 @@ public void open(DataAccessOption... options) throws IOException {
                 if (!Files.exists(getDatasetDirectory())) {
                     Files.createDirectories(getDatasetDirectory());
                 }
-                FileOutputStream fout = openLocalFileAsOutputStream();
-
-                this.setOutputStream(fout);
-                setChannel(fout.getChannel());
+//                FileOutputStream fout = openLocalFileAsOutputStream();
+//
+//                this.setOutputStream(fout);
+//                setChannel(fout.getChannel());
```

Comment:
InputStream and Channel were opened but never used cauding file descriptor leak. 
Removed the code in latest commit

Reply:
Did you also check this usage https://github.com/CeON/dataverse/blob/develop/dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/dataaccess/StorageIOUtils.java#L28 ?

Reply:
Yes. Channel is used only in case the file is opened for reading, never when opened for writing. It was a dead code.

Reply:
I see. To complete your rafactoring, I would propose to also remove both fields (`out` and `channel` and access methods) from `StorageIO` and instead add a `ReadableByteChannel readableChannel` field there. This would make the intend more explicit and prevents misuse of those fields in the future.